test/.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
```

Comment:
I'm pretty sure there's a way you can remove the duplicate code and get this based off the root `.eslintrc.js`

This doc talks about using "extends" and also about the fact that these are literally javascript files, so you might be able to work some magic related to that: https://eslint.org/docs/developer-guide/shareable-configs